src/blocks/author-list/edit.js

```
@@ -3,313 +3,259 @@
-            showDescription,
-            showSocial,
-        });
+	const loadGoogleFont = (fontName) => {
+		if (!fontName) {
+			return;
+		}
 
-        insertBlocks(gridBlock, undefined, clientId, true);
-    };
+		// Remove existing Google Fonts link if present
+		const existingLink = document.querySelector('link[href*="fonts.googleapis.com"]');
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)